coreos-postinst

```diff
@@ -114,6 +114,19 @@ EOF
     systemctl start --no-block locksmithd-kicker.timer
 fi
 
+# Azure's agent erroneously looks at the distribution name instead of its ID.
+# This prevents us from renaming the OS from "CoreOS". This patches platform.py
+# to always return "CoreOS" as the distribution name.
+PLATFORM_PATH="/usr/share/oem/python/lib64/python2.7/platform.py"
+if [ -e ${PLATFORM_PATH} ]; then
+    sum=($(md5sum ${PLATFORM_PATH}))
+    if [ ${sum} == "6315addf42c0b07f5f78d119b578e20a" ]; then
```

Comment:
Have you tested this?  It looks like you're assigning an array to `t`, and its first element will have a trailing ` /usr/share/oem...` from the `md5sum` output.  I think this is what you want:

    sum=$(md5sum ${PLATFORM_PATH})
    if [ "${sum:0:32}" == 631...